dashboard/test/controllers/omniauth_callbacks_controller/clever_test.rb

```diff
@@ -51,69 +52,9 @@ class CleverTest < ActionDispatch::IntegrationTest
       assert_credentials auth_hash, created_user
 
       assert_sign_up_tracking(
-        SignUpTracking::CONTROL_GROUP,
-        %w(
-          clever-callback
-          clever-sign-up-success
-        )
-      )
-    ensure
-      created_user&.destroy!
-    end
```

Comment:
:confounded: This diff is misleading - I just removed the two test cases below this comment.